sssom/context.py

```diff
@@ -15,14 +15,29 @@
 
 
 def get_jsonld_context():
+    """Get JSON-LD context.
```

Comment:
What JSON-LD context does it get? What's it for?

Reply:
Can you add these comments as #FIXME to the code instead of this PR? I will add the information whenever I stumble across them :)

Reply:
That's an alternative to the two possibilities I suggested in this comment https://github.com/mapping-commons/sssom-py/pull/158#discussion_r716499976. I would say we should use FIXMEs sparingly as an alternative to fixing the problem now that we found it